programs/marginfi/src/instructions/marginfi_account/sort_balances.rs

```diff
@@ -0,0 +1,20 @@
+use anchor_lang::prelude::*;
+
+use crate::{state::marginfi_account::MarginfiAccount, MarginfiResult};
+
+pub fn lending_account_sort_balances<'info>(
+    ctx: Context<'_, '_, 'info, 'info, SortBalances<'info>>,
+) -> MarginfiResult {
+    let mut marginfi_account = ctx.accounts.marginfi_account.load_mut()?;
+    marginfi_account
+        .lending_account
+        .ensure_no_gaps_in_lending_account();
+    marginfi_account.lending_account.sort_balances();
```

Comment:
Looks great! Do you think it would help to have a single bit flag to `MarginfiAccount.account_flags` and set it when sort_balances() runs, then the FE can do something like
```
if (flag){
   sort
   borrow
}else{
   borrow
}
```

or is this not helpful in any way since the sort always happens before the risk engine call anyways and consumers should always pass sorted accounts to the remaining vec now?